src/pages/paintings.js

```diff
@@ -1,8 +1,94 @@
 import * as React from "react"
+import { Composition, Box } from "atomic-layout"
+import styled from "styled-components"
 import { DefaultLayout } from "../components/DefaultLayout"
 
+import Section from "../components/Section"
+import { Button } from "../components/Button"
+import one from "../images/one.png"
+import two from "../images/two.png"
+import three from "../images/three.png"
+import four from "../images/four.png"
+
+const StyledBox = styled.div`
+  text-transform: uppercase;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  align-items: center;
+  margin-top: var(--spacing-20);
+`
+
+const StyledTitle = styled.h3`
+  color: var(--color-white);
+  font-size: var(--fontSize-3);
+  font-weight: 100;
+  text-transform: uppercase;
+  letter-spacing: var(--letter-spacing-3);
+  margin-top: 0;
+`
+
+const StyledText = styled.span`
+  color: var(--color-text-light);
+  font-size: var(--fontSize-1);
+  text-transform: capitalize;
+`
+
+const StyledImage = styled.img`
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+`
+
+const templateTablet = `
```

Comment:
This is the only template at the moment, so you can call it `imagesTemplate`, for example. No need to include "Tablet" in the name if there are no other templates. 

> Atomic Layout docs suggest names like `*Tablet` and `*Mobile` because it shows that you can have multiple templates and apply them responsively.